control-plane-operator/controllers/hostedcontrolplane/pki/konnectivity.go

```diff
@@ -32,6 +32,10 @@ func (p *PKIParams) ReconcileKonnectivityClusterSecret(secret, ca *corev1.Secret
 	return p.reconcileSignedCertWithAddresses(secret, ca, "konnectivity-server", "kubernetes", X509SignerUsage, X509UsageClientServerAuth, dnsNames, ips)
 }
 
+func (p *PKIParams) ReconcileKonnectivityClientSecret(secret, ca *corev1.Secret) error {
+	return p.reconcileSignedCertWithAddresses(secret, ca, "konnectivity-client", "kubernetes", X509SignerUsage, X509UsageClientServerAuth, nil, nil)
```

Comment:
this is a super small nit that you don't have to change and I may be wrong:

Should this just be `X509UsageClientAuth`? not clientServer

Reply:
and really be the same as the agent basically